### PR TITLE
Add Stale Bot to run daily at midnight UTC

### PR DESCRIPTION
**Pull request by PikalaxALT**
Enforce policy of closing PRs and issues with over 90 days of inactivity.

### CODE REVIEW

File: .github/workflows/stale.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+name: Close stale issues and PRs
+on:
+  schedule:
+  - cron: "0 0 * * *"
+permissions:
+  issues: write
```
Comment (red031000) [inline .github/workflows/stale.yml:6]: please exempt issues, a lot of issues are there for long term tracking

**Review by red031000 — CHANGES_REQUESTED** [review]

**Comment by RevoSucks** [comment]
Why not add a `long term` label for issues where they can be exempt by the bot?

**Comment by red031000** [comment]
also will this target your chain of PRs?

**Comment by PikalaxALT** [comment]
> also will this target your chain of PRs?

not immediately, as they all have activity within the last 60 days

**Review by red031000 — APPROVED** [review]
ok, sure, it might be good to have a label for PRs too but I understand if you may not wish to have that
pls wait to merge until the issues are labelled, which I will do when I'm able to

**Pull request merged**